scripts/gnomad-json.py

```diff
@@ -5,6 +5,66 @@
 from statsmodels.stats import proportion
 import argparse
 
+reliability = {
```

Comment:
These loci are in the TSV data file but not in the reliability dictionary:
DMD
RILPL1
XYLT1